docs/en/trust.rst

```diff
@@ -381,39 +362,55 @@ giving the references of the metadata protocol for each of these.
-|                  |                             |              |
-| Wallet Provider  | ``federation_entity``       | --           |
-|                  |                             |              |
-|                  | ``wallet_provider``         |              |
-|                  |                             |              |
-|                  |                             |              |
-+------------------+-----------------------------+--------------+
-|                  |                             |              |
-| Credential Issuer| ``federation_entity``       |              |
-|                  |                             |              |
-|                  | ``openid_credential_issuer``| `OPENID4VCI`_|
-+------------------+-----------------------------+--------------+
-|                  |                             |              |
-| Relying Party    | ``federation_entity``       |              |
-|                  |                             |              |
-|                  | ``wallet_relying_party``    | `OIDC-FED`_  |
-|                  |                             |              |
-|                  |                             | `OpenID4VP`_ |
-+------------------+-----------------------------+--------------+
+.. list-table::
+   :widths: 20 20 20 60
+   :header-rows: 1
+
+   * - OpenID Entity
+     - EUDI Entity
+     - Metadata Type
+     - References
+   * - Trust Anchor
+     - Trust Anchor
+     - ``federation_entity``
+     - `OIDC-FED`_
+   * - Intermediate
+     - Intermediate
+     - ``federation_entity``
+     - `OIDC-FED`_
+   * - Wallet Provider
+     - Wallet Provider
+     - ``federation_entity``, ``wallet_provider``
+     - --
+   * - Authorization Server
+     - 
+     - ``federation_entity``, ``oauth_authorization_server``
+     - `OPENID4VCI`_
+   * - Credential Issuer
+     - PID Provider, (Q)EAA Provider
+     - ``federation_entity``, ``openid_credential_issuer``, [``oauth_authorization_server``]
+     - `OPENID4VCI`_
+   * - Relying Party
+     - Relying Party
+     - ``federation_entity``, ``wallet_relying_party``
+     - `OIDC-FED`_, `OpenID4VP`_
+
 
 .. note::
     Wallet Provider metadata is defined in the section below.
 
     `Wallet Solution section <wallet-solution.html>`_. 
 
 
+.. note::
+    In instances where a PID/EAA Provider implements both the Credential Issuer and the Authorization Server, 
+    it MUST incorporate both 
+    ``oauth_authorization_server`` and ``openid_credential_issuer`` within its metadata types.
+    Other implementations may divide the Credential Issuer from the Authorization Server, when this happens the Credential Issuer metadata MUST contain the `authorization_servers` parameters, including the Authorization Server unique identifier.
+    Furthermore, should there be a necessity for User Authentication by the Credential Issuer,
+    it is mandatory to include the relevant metadata type, either ``openid_relying_party`` 
```

Comment:
```suggestion
    Furthermore, should there be a necessity for User Authentication by the Credential Issuer,
    it could be necessary to include the relevant metadata type, either ``openid_relying_party`` 
```